SU2_CFD/include/variables/CNEMOEulerVariable.hpp

```diff
@@ -549,4 +549,14 @@ class CNEMOEulerVariable : public CVariable {
    */
   inline unsigned short GetRhoCvveIndex(void) { return RHOCVVE_INDEX; }
   
+  /*!
+   * \brief Retrieves the value of the laminar viscosity in the primitive variable vector.
+   */
+  inline unsigned short GetLamViscIndex(void) { return LAM_VISC_INDEX; }
+
+  /*!
+   * \brief Retrieves the value of the eddy viscosity in the primitive variable vector.
+   */
+  inline unsigned short GetEddyViscIndex(void) { return EDDY_VISC_INDEX; }
+

```

Comment:
would it be possible to add this instead in the CNEMONSVariable for conceptual consistency?

Reply:
Done